iwyu.cc

```diff
@@ -2673,6 +2673,14 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     return true;
   }
 
+  bool VisitArrayType(ArrayType* type) {
+    if (CanIgnoreCurrentASTNode())
+      return true;
+    if (!compiler()->getLangOpts().CPlusPlus)
+      ReportTypeUse(CurrentLoc(), &*type->getElementType(), DerefKind::None);
```

Comment:
I think this would be clearer with a named variable;
```
const Type* elem_type = type->getElementType().getTypePtr();
ReportTypeUse(CurrentLoc(), elem_type, DerefKind::None);
```